deps.sh

```diff
@@ -385,6 +398,38 @@ elif [[ $INSTALL_ALL == false ]]; then
     fi
 
     echo "Optional dependencies: "
+    if [[ -z ${WITH_OPENVINO+x} ]] && [[ ${NO_OPENVINO+x} != true ]] && [[ "$OSTYPE" == "linux-gnu" ]]; then
+        echo -n "Do you need support for OpenVino Inference Engine? [Y/n]: "
```

Comment:
Can we switch this to 'y/N' as a default for now? I suspect, at least initially, most users won't require this as a dependency when installing from scratch